src/microbe_stage/MicrobeHUD.cs

```diff
@@ -625,6 +625,11 @@ private void UpdateHoverInfo(float delta)
             // Create for each compound the information in GUI
             foreach (var entry in compounds)
             {
+                if (entry.Value < 0.1)
```

Comment:
I'd like a comment here explaining why this is skipped, something mentioning that it is not really useful to show compounds where there is just a smidge of it there to make the hover panel less cluttered with useless stuff.